contracts/misc/TimeLocking.sol

```diff
@@ -0,0 +1,26 @@
+pragma solidity ^0.5.17;
+
+
+contract DxDAOTimeLock {
+
+    address public owner;
+    uint256 public releaseTime;
+
+    constructor(address _owner, uint256 _releaseTime) public {
+        owner = _owner;
+        releaseTime = _releaseTime;
+    }
+
+    function () external payable {
+    }
+
+    function withdraw() external {
+        require(msg.sender == owner, "only owner can withdraw");
+       // solhint-disable-next-line not-rely-on-time
+        require(releaseTime < now, "cannot withdraw before releaseTime");
+          // solhint-disable-next-line avoid-call-value
+        (bool success, ) = owner.call.value(address(this).balance)("");
```

Comment:
Why not `msg.sender.transfer(address(this).balance)` ?

Reply:
because transfer is limited in gas. it might work for dxdao avatar though no promise for other contracts which consume gas on the fallback function.
https://diligence.consensys.net/blog/2019/09/stop-using-soliditys-transfer-now/